frontend/packages/kubevirt-plugin/src/components/vm-templates/vm-template-link.tsx

```diff
@@ -1,28 +1,23 @@
 import * as React from 'react';
 import { Link } from 'react-router-dom';
 import { ResourceIcon } from '@console/internal/components/utils';
-import { getName, getNamespace, getUID } from '@console/shared';
 import { TemplateModel } from '@console/internal/models';
-import { TemplateKind } from '@console/internal/module/k8s';
 
-export const VMTemplateLink: React.FC<VMTemplateLinkProps> = ({ template }) => {
-  const name = template && (template.name || getName(template));
-  const namespace = template && (template.namespace || getNamespace(template));
-
-  return (
-    <>
-      <ResourceIcon kind={TemplateModel.kind} />
-      <Link
-        to={`/k8s/ns/${namespace}/vmtemplates/${name}`}
-        title={getUID(template)}
-        className="co-resource-item__resource-name"
-      >
-        {name}
-      </Link>
-    </>
-  );
-};
+export const VMTemplateLink: React.FC<VMTemplateLinkProps> = ({ name, namespace, uid }) => (
+  <>
+    <ResourceIcon kind={TemplateModel.kind} />
+    <Link
+      to={`/k8s/ns/${namespace}/vmtemplates/${name}`}
+      title={uid}
```

Comment:
I would just remove this. Showing the UID on hover isn't really helpful. I know we do this other places, but we're getting away from this in console.

Reply:
> I would just remove this. Showing the UID on hover isn't really helpful.

Don't worry about it for this PR.

Reply:
ok, I will push a follow-up once this gets merged